Draft of cancellation announcement (#1578)
writethedocs/www: docs/conf/australia/2021/news/cancel-announcement.rst

:template: {{year}}/generic.html

.. post:: August 25, 2021
   :tags: {{shortcode}}-{{year}}, announcement, conference

Cancelling WTD Australia and India 2021 conference
===================================================

> G'day folks, we have decided to cancel our WTD Australia and India conference in 2021.

Reviewer: ```suggestion
G'day folks. We're writing to let you know that we have decided to cancel our WTD Australia and India conference for 2021.
```

> While this hasn't been the easiest of decisions, we recognise the immense mental fatigue that lockdowns have had in Australia this year.

Reviewer 2: ```suggestion
While this has been an extremely difficult decision, we recognise the immense mental fatigue that lockdowns have caused in Australia and India this year.
```

> While this has been an extremely difficult decision, we recognise the immense mental fatigue that lockdowns have caused in Australia and India this year.

Reviewer: ```suggestion
This has been an extremely difficult decision, but we recognise the immense mental fatigue that the pandemic in general and lockdowns specifically have caused in Australia and India this year.
```

> Overall, while the community engagement has been fantastic across our remote meetup events, we are have all felt stressed and burnt out at some point these last couple of years.

Reviewer 2: ```suggestion
While the community engagement and support have been fantastic across our remote meetup events and last year's conference, we are have all felt stressed and burnt-out at various times during last couple of years.
```

> While the community engagement and support have been fantastic across our remote meetup events and last year's conference, we have all felt stressed and burnt-out at various times during the last couple of years.

Reviewer: ```suggestion
While the community engagement and support have been fantastic across our remote meetup events and last year's conference, we all felt stressed and burnt-out at various times during the last couple of years.
```

> Keeping in mind the long(er)-term health of the community, we have decided to do something else in place of the conference.

Reviewer: ```suggestion
Keeping in mind the long(er)-term health of the community, we have decided to do something else in place of the conference, in a smaller scope that will be more manageable to the organizing team while still providing some continuity to the community. 
```

Here's what we will do instead.

> Mega meetup

Reviewer: Maybe "mega" isn't the word that we want to use externally? I don't have a better suggestion....

Reviewer 3: Super?

Reviewer 4: I like the alliteration, but don't feel strongly about it.

Author: I am ok with the word "Super". Let me change that.

------------

> In place of a full 2-day conference, we will select 4 talks (from the CFP process) and invite these documentarians to present their talks at a mega 2-hour meetup (date **TBA**). 

Reviewer 2: Do you want to be less specific about this before talking to speakers?

Reviewer: ```suggestion
In place of a full 2-day conference, we will select 4 talks (from the CFP process, pending confirmation from the speakers) and invite these documentarians to present their talks at a mega 2-hour meetup (date **TBA**). 
```

We value the time and effort our documentarians put in to submit talk proposals and wanted to use this time to share their experience and knowledge.

> * This meetup event will be a non-ticketed event, so you can join in from anywhere, at any time. We will try and keep this in a timezone convenient for both our WTD Australia and India communities.

Reviewer 2: ```suggestion
* This meetup event will be a non-ticketed event, so you can join in from anywhere. We will try and keep this in a timezone convenient for both our WTD Australia and India communities.
```

> * This meetup event will be a non-ticketed event, so you can join in from anywhere. We will try and keep this in a timezone convenient for both our WTD Australia and India communities.

Reviewer: ```suggestion
* This meetup event will be a free and open event without prior registration, so you can join in from anywhere. We will try and keep this in a timezone convenient for both our WTD Australia and India communities.
```

* We will be using the WTD Australia and India Meetup groups to announce the date for this, along with the relevant WTD Slack channels and other social media.

Time to recover
----------------

2021 has been a great year with our remote meetup events in Australia.
We hope you take this time to recover and refresh yourselves mentally and physically.
We will be back in 2022 with the conference (onsite hopefully!).

Some conference housekeeping
-----------------------------

> * If you have already brought a ticket for the conference, we will refund these promptly.

Reviewer: ```suggestion
* If you already registered a ticket for the conference, we will refund these promptly and in full, without cancellation fees.
```

* If you had blocked these dates in your calendar, we will try and organise the mega meetup on one of these days, so that you do not have to take additional time off from work.

> * For those who have submitted a talk proposal, we thank you greatly and invite you to resubmit your talks for our conferences (Portland, Prague, Australia/India) next year.

Reviewer: ```suggestion
* For those who submitted a talk proposal, we thank you greatly and invite you to resubmit your talks for our conferences (Portland, Prague, Australia/India) next year.
```

> * We will be closing the CFP process over the weekend, and also cancel future ticket sales and publish this announcement on the conference website, along with the relevant Write the Docs Slack channels and other social media.

Reviewer: ```suggestion
* We will be closing the CFP process before the weekend, cancel future ticket sales, and publish this announcement on the conference website, along with the relevant Write the Docs Slack channels and other social media.
```

> * For those who submitted a talk proposal, we thank you greatly and invite you to resubmit your talks for our conferences (Portland, Prague, Australia/India) next year.

Reviewer 3: ```suggestion
* For those who submitted a talk proposal, we thank you greatly and invite you to resubmit your talks for our conferences (Portland, Prague, Australia/India) next year. If we have selected your talk for the mega meetup, we will contact you within the next two weeks.
```
(sticking with mega meetup term here for consistency, I agree with Mikey that "mega" is not my favourite term)
